golden-tests/expected-output

```diff
@@ -0,0 +1,1014 @@
+- /nix/store/5kwdw9nynm8195n9dvn6zzx0rbxb2cvm-nixos-system-nixos-22.11.20221013.cfea568.drv:{out}
```

Comment:
Does the upstream master `nix-diff` version produce the same output?

Reply:
Yes, I tested on version from nixpkgs

Reply:
Also, does this test cover all possible diff elements, i.e. there are different inputs, outputs, environment parts?

Reply:
> I'm not sure about that.

I think it's important to have tests that cover all possible cases